website/pages/docs/connect/wan_federation_via_mesh_gateways.mdx

```diff
@@ -0,0 +1,217 @@
+---
+layout: docs
+page_title: Connect - WAN Federation via Mesh Gateways
+sidebar_title: WAN Federation via Mesh Gateways <sup> Beta </sup>
+description: |-
+  WAN federation via mesh gateways allows for Consul servers in different datacenters to be federated exclusively through mesh gateways.
+---
+
+# WAN Federation via Mesh Gateways <sup>Beta</sup>
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [mesh gateways](/docs/connect/mesh_gateway).
+
+WAN federation via mesh gateways allows for Consul servers in different datacenters
+to be federated exclusively through mesh gateways.
+
+When setting up a
```

Comment:
Should we add a link (if you're using Kubernetes, see /docs/k8s/installation/multi-cluster)